zulipterminal/model.py

```diff
@@ -58,11 +58,11 @@ class ServerConnectionFailure(Exception):
     pass
 
 
-def sort_streams(streams: List[List[str]]) -> None:
+def sort_streams(streams: List[StreamData]) -> None:
```

Comment:
I'm not sure if this could be used to fix the test issue, or whether we should test it differently.